source/index.html.md

```diff
@@ -792,6 +798,47 @@ curl -X GET \
   -H "Authorization: Bearer {TOKEN}"
 ```
 
+## FileClusterScore Endpoint
+Files that have been sent for clustering, can have their estimated parsing accuracy retrieved via API.
+
+The following endpoint is currently available:
+
+| Endpoint                         | Verb  | Description                                                                       |
+| -------------------------------- | ----- | --------------------------------------------------------------------------------- |
+| /FileClusterScore/{id} | GET | Retrieve a file's estimated parsing accuracy for the given file ID  |
```

Comment:
I`d like to suggest this to be retrieved by fileGroup instead.

Do you think it would be possible? so the user just has to deal with fileGroupId as we also can just send that in the webhooks

I`m not sure how we`re going to build this endpoint by now, will we have this model FileClusterScore? if so that`s ok but if we don`t have this model we could expose the endpoint in the group endpoints

Reply:
Than the response could be a list which in DHL case will always be with 1 file only, but for other future case would bring the data of the whole group.

something like I guess
`
[
  {
    "id": 1,
    "clusterConfidenceScore": 0.1,
    "descMin": 0.1,
    "descFirstQtl": 0.2,
    "descMedian": 0.4,
    "descThirdQtl": 0.3,
    "descMax": 0.3,
    "liMin": 0.7,
    "liFirstQtl": 0.8,
    "liMedian": 0.8,
    "liThirdQtl": 0.9,
    "liMax": 1.0,
  },
]
`